external/CMakeLists.txt

```diff
@@ -77,6 +77,7 @@ target_compile_options(ParallelSTL INTERFACE -Wno-unused-parameter
                                            -Wno-sign-compare
                                            -Wno-reorder
                                            -Wno-unused-local-typedefs
+                                           -Wno-unused-variable
```

Comment:
Why?

Reply:
ParallelSTL generates some warnings. I am only limiting this to ParallelSTL's code. 
Ideally we should submit a PR to ParallelSTL's repository. I have done that many times and they take forever to accept a PR. So I prefer to ignore for now

Reply:
Ok.